lib/RAST_SDK/RAST_SDKImpl.pm

```diff
@@ -755,6 +760,10 @@ sub annotate {
 
 	#print Bio::KBase::utilities::to_json($contigobj,1));
 	#print Bio::KBase::utilities::to_json($genome,1);
+	use JSON;
+	open my $fh, ">", "/kb/module/work/tmp/data_out.json";
+	print $fh encode_json($genome);
+	close $fh;
```

Comment:
I assume this is what is generating the genome output.  How was this working before.  I see the GFU call in the next line.  Where was the input coming from before?

Reply:
This was saving a copy to scratch for my debugging. Will remove.